src/frontend/komponenter/Felleskomponenter/HeaderMedSøk/OpprettFagsakModal.tsx

```diff
@@ -95,7 +97,19 @@ const OpprettFagsakModal: React.FC<IOpprettFagsakModal> = ({
     );
     const [visFeilmelding, settVisFeilmelding] = useState(false);
     const [valgtSamhandler, settValgtSamhandler] = useState<ISamhandlerInfo | undefined>(undefined);
-    const { onSubmitWrapper, samhandlerSkjema } = useSamhandlerSkjema();
+    const [spinner, settSpinner] = useState(false);
+    const { onSubmitWrapper, samhandlerSkjema } = useSamhandlerSkjema(
+        _r => {
+            settSpinner(false);
+            settFeilmelding('');
+            settVisFeilmelding(false);
+        },
+        _r => {
+            settFeilmelding('Finner ikke institusjon. Kontakt NØS for å opprette TSS-ident.');
```

Comment:
Burde ikke denne feilmeldingen komme fra backend med en FrontendFeil når man faktisk ikke finner en TSS-ident? For slik så blir vel alle feilene mot samhandler markert som kontakt NØS?